### DISCUSSION TITLE
Zigbee devices are active but no connections shown on map

### DISCUSSION THREAD

**Discussion by nmaou** [arendst/Tasmota#20061] [Q&A]
Hello,
My network consists of a ZBBridge coordinator, 2 ikea led lamps which work as routers, an ikea zigbee repeater
 and several sonoff zigbee end devices.
![zbmap](https://github.com/arendst/Tasmota/assets/8890082/ac971a69-8097-401f-946c-c00005f659c0)
All devices are active but the map shows no connections from the devices to a router or the coordinator.
My problem is that I want to use zbping to chech is a device is alive and connected to the network but it does not work. I returns "reachable" = "false" for all devices that in the map are not shown connected to a router or the coordinator.
Any comments are welcomed.

**Comment by s-hadinger** [comment]
Not being registered in a router is unexpected but can happen. Maybe one of the router is not reporting its children. 

You can't ping a device if it's sleeping which is the case for battery powered devices

**Reply by sfromis** [reply]
If the times shown per device on the Tasmota front page are "too long ago", there may be cause for concern.
